realm/realm-library/src/main/java/io/realm/internal/util/Pair.java

```diff
@@ -27,8 +27,8 @@
  * objects.
  */
 public class Pair<F, S> {
-    public final F first;
-    public final S second;
+    public F first;
```

Comment:
Why this cannot be `final` anymore? I cannot find code changing it after construction.